repository/lib/rom/repository/class_interface.rb

```diff
@@ -113,7 +113,13 @@ def define_restricted_command_method(type, views, **opts)
             if changeset.respond_to?(:commit)
               changeset.commit
             else
-              root.command(type, **opts).public_send(view_name, *view_args).call(*input)
+              #   in case of view_args been coercible to an array we do not want to splat it
+              #   example: Date and Time objects
+              if view_args.respond_to?(:to_a)
+                root.command(type, **opts).public_send(view_name, view_args).call(*input)
```

Comment:
I'll wrap this up folks, it's pretty clear this is a bit more challenging to fix.